libV2/schemaUtils.js

```diff
@@ -246,9 +259,10 @@ let QUERYPARAM = 'query',
    * @returns {Object} Returns the object that staisfies the schema
    */
   resolveRefFromSchema = (context, $ref, stackDepth = 0, seenRef = {}) => {
-    const { specComponents } = context;
+    const { specComponents } = context,
+      { stackLimit } = context.computedOptions;
```

Comment:
What is the source of `computedOptions`?

Reply:
@dhwaneetbhatt It gets calculated upon defining the SchemaPack constructor. This constructor is used before all API calls.

https://github.com/postmanlabs/openapi-to-postman/blob/b4ef1dc0caf1f899dd0b2272d8932c75ddda552b/lib/schemapack.js#L61

Reply:
Also `context` is same as `this` of the constructor.

https://github.com/postmanlabs/openapi-to-postman/blob/b4ef1dc0caf1f899dd0b2272d8932c75ddda552b/lib/schemapack.js#L98